selfdrive/car/volkswagen/carstate.py

```diff
@@ -60,6 +60,12 @@ def update(self, pt_cp):
     # We use the speed preference for OP.
     self.displayMetricUnits = not pt_cp.vl["Einheiten_01"]["KBI_MFA_v_Einheit_02"]
 
+    # Consume blind-spot radar info/warning LED states, if available
+    ret.leftBlindspot = any([bool(pt_cp.vl["SWA_01"]["SWA_Infostufe_SWA_li"]),
```

Comment:
can you add a comment explaining the info and warning states?

Reply:
since it's only two signals, I'd also just use an or instead of building a list and using any

Reply:
Sure, NP